spicy/runtime/src/util.cc

```diff
@@ -29,6 +29,16 @@ std::string spicy::rt::bytes_to_hexstring(const hilti::rt::Bytes& value) {
     return result;
 }
 
+std::string spicy::rt::bytes_to_mac(const hilti::rt::Bytes& value) {
+    std::vector<std::string> bytes;
+    bytes.reserve(value.size());
+
+    for ( auto x : value )
+        bytes.emplace_back(hilti::rt::fmt("%02X", x));
+
+    return hilti::rt::join(bytes, "::");
```

Comment:
Total flyby: Zeek formats `endpoint$l2_addr` with a single `:` (and also has special logic for discarding the last two bytes of 8byte mac's ending with `\x00\x00`). Can see being consistent or configurable with `::` vs `:` useful.

https://github.com/zeek/zeek/blob/96c091f7acac53ccf64a28c6221b3830fd2bf439/src/net_util.cc#L220

We've also previously found that calling `snprintf()/fmt()` for individual bytes results noticeable overhead (it's done for every new ConnVal twice by default). Unsure that matters all that much here, but think `rt::fmt` is pretty heavy.